docs/5.0/pages/database-access.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Database Access
+description: Teleport Database Access introduction, demo and resources
+---
+
+# Database Access
+
+Teleport can provide secure access to PostgreSQL and MySQL databases, while
+improving both access control and visibility.
+
+Some of the things you can do with Database Access include:
+
+* Allow users to retrieve short-lived database credentials using single sign-on
+  flow thus maintaining their organization-wide identity.
+* Configure rich role-based access control policies for databases and implement
+  custom [access workflows](./enterprise/workflow/index.mdx).
+* Capture database access events as well as query activity in the audit log.
+
+## Demo
+
+In the following demo video we connect to a PostgreSQL server with `psql` and
+pgAdmin 4 after authenticating with Github, execute a few SQL queries and
+observe them in the audit log.
+
+<video autoPlay loop muted playsInline controls>
+  <source src="https://goteleport.com/teleport/videos/database-access-preview/dbaccessdemo.mp4" type="video/mp4" />
+  <source src="https://goteleport.com/teleport/videos/database-access-preview/dbaccessdemo.webm" type="video/webm" />
+Your browser does not support the video tag.
+</video>
+
+## Getting Started
+
+Configure Database Access from scratch in a 10 minute [Getting Started](./database-access/getting-started.mdx)
+guide.
+
+## Guides
+
+The following guides are available for configuring supported databases and
+deployments:
+
+* [AWS RDS/Aurora PostgreSQL](./database-access/guides/postgres-aws.mdx)
+* [AWS RDS/Aurora MySQL](./database-access/guides/mysql-aws.mdx)
+* [Self-hosted PostgreSQL](./database-access/guides/postgres-self-hosted.mdx)
+* [Self-hosted MySQL](./database-access/guides/mysql-self-hosted.mdx)
```

Comment:
As we add more databases I would suggest we group by type -> deployment type. e.g. Postgres -> Self-hosted vs AWS RDS, GCP etc

Reply:
@benarent You mean change the order of the entries? This is actually intentional, so the most popular use-cases (which RDS/Aurora are) appear first.